pennylane/transforms/decompositions/two_qubit_unitary.py

```diff
@@ -14,227 +14,202 @@
 """Contains transforms and helpers functions for decomposing arbitrary two-qubit
 unitary operations into elementary gates.
 """
-import warnings
-
 import pennylane as qml
 from pennylane import numpy as np
 from pennylane import math
 
 from .single_qubit_unitary import zyz_decomposition
+from .utils import (
+    _convert_to_su4,
+    _compute_num_cnots,
+    _su2su2_to_tensor_products,
+    _extract_su2su2_prefactors,
+    E,
+    Edag,
+    LAST_COL_NEG,
+)
```

Comment:
I moved them mostly just to clean up this file to make things easier to review :sweat_smile: I could definitely move them back, since I don't think they are useful anywhere else right now.